### PR TITLE
Re-enable live gmail tests

### LINKED ISSUE #3929: messages on live test gmail account missing
This may possibly be caused by one of the updates that deletes last message in conversation - or similar - possibly it could have gradually deleted all messages by mistake?

image

In this issue, let's try to identify the offending code and remove it.

Then in a follow up PR me or someone will re-populate the inbox and do a backup of it somehow before we run more tests.

Or maybe we ensure the emails are present (or upload them) before each test.

### PR DESCRIPTION
**Pull request by limonte**
This PR re-enables 4 live gmail tests that were broken (#3929)

issue #3929

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities
- [ ] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Comment by limonte** [comment]
@rrrooommmaaa In this PR I re-enabled 4 tests, there are still 5 of them left to re-enable, but that is blocked by [this](https://github.com/FlowCrypt/flowcrypt-browser/issues/3929#issuecomment-907859243).

Let's review and merge this PR, and the rest of the tests I will re-enable in another one.

**Comment by limonte** [comment]
> @rrrooommmaaa In this PR I re-enabled 4 tests, there are still 5 of them left to re-enable, but that is blocked by [this](https://github.com/FlowCrypt/flowcrypt-browser/issues/3929#issuecomment-907859243).
> 
> Let's review and merge this PR, and the rest of the tests I will re-enable in another one.

Sorry, I just noticed the email from Tom which will help me to re-enable the rest of the tests. I will continue working on this PR.

**Comment by limonte** [comment]
Ready for review 🚀

1. While re-enabling Thunderbird tests I faced 2 issues which are reported in #3943 and #3944
2. I merged two "Thunderbird [html]" tests into 1 for the sake of tests speed, those two tests were using the same message for testing.
3. I merged two "Thunderbird [plain]" tests into 1 for the sake of tests speed, those two tests were using the same message for testing.

File: extension/js/common/lang.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export const Lang = { // tslint:disable-line:variable-name
...
     yourKeyCantOpenImportIfHave: 'Your current key cannot open this message. If you have any other keys available, you should import them now.\n',
     encryptedCorrectlyFileBug: 'It\'s correctly encrypted for you. Please file a bug report if you see this on multiple messages. ',
-    singleSender: 'Normally, messages are encrypted for at least two people (sender and the receiver). It seems the sender encrypted this message manually for themselves, and forgot to add you as a receiver. This sometimes happens when the sender is using OpenPGP software other than FlowCrypt, because they have to configure encryption manually, and mistakes can happen.',
+    singleSender: 'Normally, messages are encrypted for at least two people (sender and the receiver). It seems the sender encrypted this message manually for themselves, and forgot to add you as a receiver. This sometimes happens when the sender is using OpenPGP software other than FlowCrypt, because they have to configure encryption manually, and mistakes can happen. ',
```
Comment (limonte) [inline extension/js/common/lang.ts:40]: tiny fix for this:

CleanShot 2021-08-30 at 21 42 49@2x

**Comment by rrrooommmaaa** [comment]
I guess some of lost messages may be present in `exported-messages` folder. It would be good to add these new Gmail messages there (or to another folder (or in EML format)) and add remarks in the tests with their message ids (file names) for easier restoration next time.

**Comment by limonte** [comment]
> I guess some of lost messages may be present in `exported-messages` folder. It would be good to add these new Gmail messages there (or to another folder (or in EML format)) and add remarks in the tests with their message ids (file names) for easier restoration next time.

Done, thanks 👍  I did backup in this format: `test/source/mock/google/exported-messages/live-gmail-test-emails/{EMAIL_SUBJECT} ({EMAIL_ID}).eml`.

**Comment by limonte** [comment]
Tests are failing with the reason that seems to be unrelated to the PR:

CleanShot 2021-08-31 at 15 17 57@2x

@rrrooommmaaa @martgil  could that be related to #3942 even though it's not merged yet?

**Comment by rrrooommmaaa** [comment]
yes, `exported-messages` are parsed automatically (JSONs files are expected to be there), so presence of a directory there makes it fail. Can you move this `live-gmail...` directory up one level.

**Comment by limonte** [comment]
> yes, `exported-messages` are parsed automatically (JSONs files are expected to be there), so presence of a directory there makes it fail. Can you move this `live-gmail...` directory up one level, please?

Oh, didn't expect that, thanks for the hint!

**Comment by limonte** [comment]
All good now, ready for review 🚀

btw, this 

> 2. I merged two "Thunderbird [html]" tests into 1 for the sake of tests speed, those two tests were using the same message for testing.
> 3. I merged two "Thunderbird [plain]" tests into 1 for the sake of tests speed, those two tests were using the same message for testing.

saved around 1 min of running time for live gmail tests

**Review by rrrooommmaaa — APPROVED** [review]

**Pull request merged**